utils/parts.stanza

```diff
@@ -971,7 +971,9 @@ defn Model3D (json: JObject) -> Model3D :
   Model3D(json["filename"] as String,
           Vec3D(json["position"] as JObject),
           Vec3D(json["scale"] as JObject),
-          Vec3D(json["rotation"] as JObject))
+          Vec3D(json["rotation"] as JObject),
+          get(json, "properties") as Tuple<KeyValue<String, String>>,
```

Comment:
I don't know this is possible.

Reply:
I wasn't sure either, and I started using the approach from `VendorPartNumbers` - but, in testing, this one seemed to work.

Reply:
Update - although it worked within OCDB, it failed later on during serialization.

Have updated to remove the `properties`, and pass the new comment we need as a `comment` field.